gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,29 @@
-"""Add docstring."""
+"""This module is used for storing the main functions requested from the Github Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output, and colorization of text."""
 
+import gator
+import os
+from output import output_tools
+from typing import List, Tuple
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+# Commands are received as tuple of lists with lists
+# The tuple is used for future upgradeability for when shell commands are introduced
+# The string is the path to the file or filename and the list is the commands for the file.
+def receive_command(commands_input):

```

Comment:
I think it's important to be careful how we name the functions here. This one does more than just receiving the command. It also runs the commands and gets the results. So, you might want to revise the name of the function to accurately reflect what it does.